### PR TITLE
.bat => .sh

### PR DESCRIPTION
**Pull request by AngelGryph**
Linux version of mod packaging script.  The shell script assumes the user has tar, gzip and rar installed and accessible through $PATH.  Should be used with a version of ModPackaging that also has the Linux shell scripts.

### CODE REVIEW

File: package_mod.sh [hunk 1]
```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
+
+# /* this initializes the values for the different variables referenced in compatible_games */
+bash "../ModPackaging/utilities/ie_games.sh"
```
Comment (lynxlynxlynx) [inline package_mod.sh]: I'm surprised this works as expected, as it starts a subshell. Typically one would use source or . here.

Comment (AngelGryph) [reply package_mod.sh]: Good point, I need to fix that one.  Will do so asap.

File: package_mod.sh [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+source "../ModPackaging/utilities/ie_games.sh"
+
+# /* MODIFY: set the values of the 3 variables below to reflect the current mod version */
+export mod_name="Real Mod"
+export mod_version="v2"
+export mod_folder="real_mod"
+
+# /* MODIFY: list here which IE games the mod is compatible with, from this list of possibilities: */
+# /* ${bg1}, ${bg2}, ${bgt}, ${tutu}, ${bgee}, ${bgiiee}, ${iwd1}, ${iwdee}, ${iwd2}, ${iwd2ee}, ${iwd-in-bg2}, ${pst}, ${pstee}, ${eet} */
+export compatible_games="${bg1}, ${bg2}, ${bgt}, ${tutu}, ${bgee}, ${bgiiee} or ${eet}"
```
Comment (lynxlynxlynx) [inline package_mod.sh]: this looks unused and would only work in the readme.

Comment (AngelGryph) [reply package_mod.sh]: It is, currently.  The .bat file uses it for including some text in the self-extracting rar.  I'll check if something like that can be done with zip files.  And if not, then I guess we can eliminate this whole blurp for Linux.

Comment (AngelGryph) [reply package_mod.sh]: Sorry it took me so long to get back on this one. The unused variable has been removed now.

**Comment by mike1072** [comment]
Is this ready to be merged now?

**Comment by AngelGryph** [comment]
It is as far as I am concerned.  I updated the branch to be in sync with the current master, all it does is add the Linux bash file.  Everything else is unchanged.

**Comment by mike1072** [comment]
Great, I'll merge it tomorrow unless I hear any objections from @lynxlynxlynx or anyone else.

**Review by lynxlynxlynx — APPROVED** [review]

**Pull request merged**